### DISCUSSION TITLE
"device connected" but once i go back to my IDE it isn't connected

### DISCUSSION THREAD

**Discussion by candyppn** [community/community#38816] [Projects and Issues]
### Select Topic Area

Bug

### Body

I installed copilot with IntelliJ marketplace but once i login, i have the link and everything that tell me that i'm connected but when i go back to IntelliJ i'm not .
tried to deinstall copilot and IntelliJ, restart intellij but i can't seem to use copilot on intellij.

**Comment by ossanna16** [comment]
Hi there @candyppn and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
